packages/next-swc/crates/core/tests/full/example/output.js

```diff
@@ -1,40 +1,40 @@
-function a(a, b) {
-    (null == b || b > a.length) && (b = a.length);
-    for(var c = 0, d = new Array(b); c < b; c++)d[c] = a[c];
+function b(c, a) {
```

Comment:
This is changed because the name mangler now give higher priorities to identifiers more frequently used.